src/main.js

```diff
@@ -0,0 +1,162 @@
+const selectors = {
+    INPUT: '.keyboard-input',
+    CHAR_KEY: '.char',
+    ACTIVATABLE_KEY: '.activatable-key',
+};
+
+const classes = {
+    KEYBOARD: 'keyboard',
+    HIDDEN: 'hidden',
+    KEYBOARD_KEYS: 'keyboard-keys',
+    KEY: 'key',
+    WIDE_KEY: 'wide-key',
+    ACTIVATABLE_KEY: 'activatable-key',
+    EXTRA_WIDE_KEY: 'extra-wide-key',
+    DARK_KEY: 'dark-key',
+    CHAR_KEY: 'char',
+    ACTIVE_KEY: 'active-key',
+};
+
+const keyboardInputElement = document.querySelector(selectors.INPUT);
+let capsLock = false;
+
+const keysHandlers = {
+    backspace: () => {
+        keyboardInputElement.value = keyboardInputElement.value.slice(0,keyboardInputElement.value.length - 1);
+    },
+    keyboard_return: () => {
+        keyboardInputElement.value += '\n';
+    },
+    check_circle: () => {
+        keyboardElement.classList.add(classes.HIDDEN);
+    },
+    space_bar: () => {
+        keyboardInputElement.value += ' ';
+    },
+    keyboard_capslock: () => {
+        const buttons = document.querySelectorAll(selectors.CHAR_KEY);
+        const capsButton = document.querySelector(selectors.ACTIVATABLE_KEY);
+        
+        capsLock = !capsLock;
+        buttons.forEach(el => {
+            el.textContent = capsLock ? el.textContent.toUpperCase() : el.textContent.toLowerCase();
+        });
+        capsButton.classList.toggle(classes.ACTIVE_KEY);
+    },
+
+    default: (e) => {
+        keyboardInputElement.value += e.target.innerHTML;
+    }
+
+};
+
+let keyboardElement;
+let keyContainerElement;
+
+const createIconHTML = (icon_name) => {
```

Comment:
createHtmlIcon or createIconInHtml
Please do not forget about dash case icon-name